lib/std/priority_queue.zig

```diff
@@ -86,6 +89,23 @@ pub fn PriorityQueue(comptime T: type) type {
             return first;
         }
 
+        pub const Error = error{BoundsError};
+
+        /// Remove and return element at index. If there aren't enough
+        ///  elements to remove from idx, returns null. Indices are in
+        ///  the same order as iterator, which is not necessarily
+        ///  priority order.
+        pub fn removeIdx(self: *Self, idx: usize) !T {
```

Comment:
We have precedence in the std lib for spelling "Index" out:

https://github.com/ziglang/zig/blob/9390e8b84883331757d3ce15cfca89279aceb090/lib/std/segmented_list.zig#L239

Reply:
Also a good point.